actions/users.ts

```diff
@@ -0,0 +1,23 @@
+"use server";
+
+import { checkClerkId, createUser, getOneUser, getUsers } from "@/lib/users";
+
+export async function actionGetUsers() {
+  const users = await getUsers();
+  return users;
+}
+
+export async function actionGetOneUser(id: string) {
```

Comment:
```suggestion
export async function actionGetUser(id: string) {
```

I would rename the method to `actionGetUser`, the shorter we can go with the naming the better :)